src/components/20-molecules/datepicker/index.scss

```diff
@@ -182,36 +183,45 @@ axa-datepicker {
   }
 
   &__weekdays {
-    display: flex;
-    margin: 0 -30px;
-    padding-left: 30px;
-    padding-right: 30px;
-    flex-direction: row;
-    justify-content: space-between;
     background-color: $color-axa-blue;
     color: $color-prim-white;
+    margin: 0 -100%;
```

Comment:
🤔

Reply:
I don't know of another way to spread the background stripe to the edges, as per the design. Suggestions welcome.